cplusplus/check_square.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+
+using namespace std;
+
+void issquare(long long int a){
+int flag=1;	for(long long  i=1;i<=a/2+1;i++){	if(i*i==a)	{cout<<"it is a square number\n";flag=0;}	}if(flag==1)	cout<<"Not a square number\n";		
```

Comment:
flag isn't modified and this whole if statement for cout can be simplified down to one line via ```c
cout << condition ? <const char*>1 : <const char*>2```